guides/micronaut-jaxrs-jdbc/micronaut-jaxrs-jdbc.adoc

```diff
@@ -0,0 +1,94 @@
+common:header.adoc[]
+
+The application exposes some REST endpoints using https://projects.eclipse.org/projects/ee4j.jaxrs[JAX-RS annotations] and stores data in a MySQL database using Micronaut Data JDBC.
+
+common:requirements-mysql-testcontainers.adoc[]
+
+common:completesolution.adoc[]
+
+common:create-app-features.adoc[]
+
+common:datasource-configuration.adoc[]
+
+common:flyway.adoc[]
+
+resource:db/migration/V1__schema.sql[]
+
+During application startup, Flyway will execute the SQL file and create the schema needed for the application.
+
+=== Domain
+
+Create the domain entities:
+
+source:Pet[]
+
+callout:mapped-entity[1]
+callout:mapped-entity-id[2]
+callout:generated-value[3]
+
+The above entity uses an enum:
+
+source:PetType[]
+
+=== Service
+
+Create a POJO `NameDto`:
+
+source:NameDto[]
+
+callout:introspected[1]
+
+Create a https://micronaut-projects.github.io/micronaut-data/latest/guide/#dbcRepositories[Repository]:
+
+source:PetRepository[]
+
+callout:jdbcrepository[1]
+callout:crudrepository[2]
+callout:dto-projection[3]
+
+=== JAX-RS
+
+==== Dependencies
+
+When you add `jax-rs` feature, the generated application includes the following dependencies:
+
+:dependencies:
+dependency:micronaut-jaxrs-processor[groupId=io.micronaut.jaxrs,scope=annotationProcessor]
+dependency:micronaut-jaxrs-server[groupId=io.micronaut.jaxrs]
+:dependencies:
+
+==== Resource
+
+Define an endpoint using JAX-RS.
+
+source:PetResource[]
+
+<1> A JAX-RS annotation to define the base URI for methods in this class.
+callout:constructor-di[number=2,arg0=PetRepository]
+<3> A JAX-RS annotation to define an endpoint for HTTP Get requests. Mapped to the base URI `/pets` since no `@Path` is provided.
+<4> The response is converted to JSON Array automatically.
+<5> Paths can be templated for path parameters, such as the pet's name here.
+<6> Associate the template placeholder name with the Java variable.
+
+=== Tests
+
+Add a test for the resource:
+
+test:PetResourceTest[]
+
+callout:micronaut-test-transactional-false[1]
+callout:http-client[2]
+<3> Micronaut returns a 404 response if the resource method returns an empty optional
+
+common:testApp.adoc[]
+
+common:runapp-mysql-with-docker.adoc[]
+
+common:graal-with-plugins.adoc[]
+
```

Comment:
repeat the usage info (TBD) from the run app section here so they can see that everything works that same with native image